doc/kubernetes/modules/ROOT/partials/aurora/aurora-create-peering-connections.adoc

```diff
@@ -0,0 +1,214 @@
+. Retrieve the Aurora VPC
++
+.Command:
+[source,bash]
+----
+aws ec2 describe-vpcs \
+  --filters "Name=tag:AuroraCluster,Values=keycloak-aurora" \
+  --query 'Vpcs[*].VpcId' \
+  --region eu-west-1 \
+  --output text
+----
++
+.Output:
+[source,json]
+----
+vpc-0b40bd7c59dbe4277
+----
++
+. Retrieve the ROSA cluster VPC
+.. Login to the ROSA cluster using `oc`
+.. Retrieve the ROSA VPC
++
+.Command:
+[source,bash]
+----
+NODE=$(oc get nodes --selector=node-role.kubernetes.io/worker -o jsonpath='{.items[0].metadata.name}')
+aws ec2 describe-instances \
+  --filters "Name=private-dns-name,Values=${NODE}" \
+  --query 'Reservations[0].Instances[0].VpcId' \
+  --region eu-west-1 \
+  --output text
+----
++
+.Output:
+[source,json]
+----
+vpc-0b721449398429559
+----
++
+. Create Peering Connection
++
+.Command:
+[source,bash]
+----
+aws ec2 create-vpc-peering-connection \
+  --vpc-id vpc-0b721449398429559 \# <1>
+  --peer-vpc-id vpc-0b40bd7c59dbe4277 \# <2>
+  --peer-region eu-west-1 \
+  --region eu-west-1
+----
+<1> ROSA cluster VPC
+<2> Aurora VPC
++
+.Output:
```

Comment:
This and other outputs are very verbose. It seems we need only part of the output for the later commands. Could we perhaps shorten them wherever it is possible?

Reply:
That's true. I was torn between providing detailed output for completeness and providing succinct docs. One way we could improve this would be to add `--query` args to some of the commands where only a single value is required, that way the output will be much less verbose.